install_config/cluster_metrics.adoc

```diff
@@ -753,3 +753,291 @@ by performing the following steps:
+  openshift_prometheus_pv_data:
+  -  pv_name: prometheus
+     pv_template: prom-pv-server.yml
+     pv_label: Prometheus Server PV
+  -  pv_name: prometheus-alertmanager
+     pv_template: prom-pv-alertmanager.yml
+     pv_label: Prometheus Alertmanager PV
+  -  pv_name: prometheus-alertbuffer
+     pv_template: prom-pv-alertbuffer.yml
+     pv_label: Prometheus Alert Buffer PV
+
+# Hostname/IP of the NFS server. Currently defaults to first master
+openshift_prometheus_nfs_server: "{{ groups.nfs.0 }}"
+
+# storage
+openshift_prometheus_storage_type: pvc
+openshift_prometheus_pvc_name: prometheus
+openshift_prometheus_pvc_size: 10G
+openshift_prometheus_pvc_access_modes: [ReadWriteOnce]
+openshift_prometheus_pvc_pv_selector: {}
+
+openshift_prometheus_alertmanager_storage_type: pvc
+openshift_prometheus_alertmanager_pvc_name: prometheus-alertmanager
+openshift_prometheus_alertmanager_pvc_size: 10G
+openshift_prometheus_alertmanager_pvc_access_modes: [ReadWriteOnce]
+openshift_prometheus_alertmanager_pvc_pv_selector: {}
+openshift_prometheus_cpu_limit: null
+openshift_prometheus_memory_limit: null
+openshift_prometheus_cpu_requests: null
+openshift_prometheus_memory_requests: null
+openshift_prometheus_alertmanager_cpu_limit: null
+openshift_prometheus_alertmanager_memory_limit: null
+openshift_prometheus_alertmanager_cpu_requests: null
+openshift_prometheus_alertmanager_memory_requests: null
+openshift_prometheus_alertbuffer_cpu_limit: null
+openshift_prometheus_alertbuffer_memory_limit: null
+openshift_prometheus_alertbuffer_cpu_requests: null
+openshift_prometheus_alertbuffer_memory_requests: null
+openshift_prometheus_oauth_proxy_cpu_limit: null
+openshift_prometheus_oauth_proxy_memory_limit: null
+openshift_prometheus_oauth_proxy_cpu_requests: null
+openshift_prometheus_oauth_proxy_memory_requests: null
+----
+
+*Prometheus Storage Related Variables*
+
+With each Prometheus component--including Prometheus,
+Alertmanager, alert-buffer, and oauth-proxy--you can set the PV claim by setting
+corresponding role variable, for example:
+
+----
+openshift_prometheus_storage_type: pvc
+openshift_prometheus_alertmanager_pvc_name: alertmanager
+openshift_prometheus_alertbuffer_pvc_size: 10G
+openshift_prometheus_pvc_access_modes: [ReadWriteOnce]
+----
+
+*Prometheus Alert Rules File Variable*
+
+You can add an external file with alert rules by setting the path to an additional
+rules variable:
+
+----
+openshift_prometheus_additional_rules_file: <PATH>
+----
+
+The file content should be in Prometheus alert rules format. The following
+example sets a rule to send an alert when one of the cluster nodes is down:
+
+----
+groups:
+- name: example-rules
+  interval: 30s # defaults to global interval
+  rules:
+  - alert: Node Down
+    expr: up{job="kubernetes-nodes"} == 0
+    annotations:
+      miqTarget: "ContainerNode"
+      severity: "HIGH"
+      message: "{{ '{{' }}{{ '$labels.instance' }}{{ '}}' }} is down"
+----
```

Comment:
Do we have a guide or does upstream have a guide on writing rules/alerts? 
Do we/Can we document what alerts Online uses?

Reply:
Same for this. We can get this added post-3.7. Thank you!